.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,7 +12,7 @@ about: Report a reproducible bug in the current release of nautobot-netbox-impor
 ### Expected Behavior
 
 
-<!-- What happened instead? -->
+<!-- What happened instead? Please include as much log data as possible. -->
```

Comment:
Should we provide a link to how to retrieve these logs?

Reply:
This is a management command so all of the output is on stdout/stderr but I updated this line to be more descriptive.